src/sentry/api/urls.py

```diff
@@ -1442,6 +1443,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         OrganizationSpansFieldValuesEndpoint.as_view(),
         name="sentry-api-0-organization-spans-fields-values",
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/spans/frequency-stats/$",
```

Comment:
We're not returning frequency, it's counts so let's just name it generic `stats`?
```suggestion
        r"^(?P<organization_id_or_slug>[^\/]+)/spans/fields/stats/$",
```